backend/api/Models/Parcel/ParcelEvaluationModel.cs

```diff
@@ -3,7 +3,7 @@ namespace Pims.Api.Models.Parcel
     public class ParcelEvaluationModel : BaseModel
     {
         #region Properties
-        public int PropertyId { get; set; }
+        public int ParcelId { get; set; }
```

Comment:
Note I made these match the database column names. I don't know if this will breach @devinleighsmith your Property Detail View Page. Additionally most of this is going to be updated/changed this sprint because we are changing how we implemented evaluations.